### ISSUE TITLE
Experiences with pybullet

### ISSUE THREAD

**Issue by SebastianGB** [DLR-RM/RAFCON#7]
_Originally posted by @chris-rogers-anki in https://github.com/DLR-RM/RAFCON/issues/4#issuecomment-426729853_

...
Has anyone combined rafcon with pybullet.org?

**Comment by SebastianGB** [comment]
@chris-rogers-anki: 
This sounds interesting! In our institute we use bullet also for robot simulation, most of the time in the Gazebo simulator. But, as far as I know, we didn't use pybullet (yet).

Can you explain your use case a little bit?

**Comment by chris-rogers-anki** [comment]
Hello,
We (Anki.com) make a consumer robot called Vector (launches next Friday)
I'm on the art tools team.  We program animation tools for our robot.  We
have a state machine editor we built, but its very simple compared to yours.
We used Webots to do the robot sims but are looking at other possibilities,
and it seems like pybullet (bullet with python bindings and a focus on
robotic sim and reinforcement learning) and rafcon are worth looking into.
